pkg/scheduler/factory_test.go

```diff
@@ -190,17 +192,29 @@ func TestCreateFromConfigWithHardPodAffinitySymmetricWeight(t *testing.T) {
 		"priorities" : [
 			{"name" : "RackSpread", "weight" : 3, "argument" : {"serviceAntiAffinity" : {"label" : "rack"}}},
 			{"name" : "NodeAffinityPriority", "weight" : 2},
-			{"name" : "ImageLocalityPriority", "weight" : 1}
+			{"name" : "ImageLocalityPriority", "weight" : 1},
+			{"name" : "InterPodAffinityPriority", "weight" : 1}
```

Comment:
What is the prioritiby between `c. hardPodAffinitySymmetricWeight` and `hardPodAffinityWeight` in plugin config, which one would be overwritten if we have them both?

Reply:
It was never documented. But looking at the appends in `Configurator.createFromConfig` and `Configurator.createFromProvider` and how the arrays are processed in `NewFramework`, it looks like the pluginConfig had higher preference. This doesn't change with this PR.

But ultimately, it doesn't matter much, as a follow up PR is removing `c.hardPodAffinitySymmetricWeight` from v1alpha2.